orator/commands/command.py

```diff
@@ -77,7 +77,7 @@ def _check_config(self):
         """
         current_path = os.path.relpath(os.getcwd())
 
-        accepted_files = ['orator.yml', 'orator.py']
+        accepted_files = ['orator.yml', 'orator.py', 'config.py', 'db.py', 'config.json', 'db.json']
```

Comment:
Agreed, but the support for JSON or a better documentation talking about configuration files for Orator would be very helpful!